panther_power_control/src/power_board_node.py

```diff
@@ -146,35 +148,40 @@ def __init__(self, name: str) -> None:
         rospy.loginfo(f'[{rospy.get_name()}] Node started')
 
     def _cmd_vel_cb(self, *args) -> None:
-        self._cmd_vel_msg_time = rospy.get_time()
+        with self._lock:
+            self._cmd_vel_msg_time = rospy.get_time()
 
     def _motor_controllers_state_cb(self, msg: DriverState) -> None:
-        self._can_net_err = any({msg.rear.fault_flag.can_net_err, msg.front.fault_flag.can_net_err})
+        with self._lock:
+            self._can_net_err = any({msg.rear.fault_flag.can_net_err, msg.front.fault_flag.can_net_err})
```

Comment:
This variable is used in `_e_stop_reset_cb`, so we should include `_lock` there. 

Additionally, we could consider using different `Lock` objects in various situations, which may help reduce reaction time in critical scenarios and enhance the _safety factor_. However, the decision is ultimately yours.